PDF_UA/WCAG-21-Complete.xml

```diff
@@ -6,7 +6,7 @@
     </details>
     <hash></hash>
     <rules>
-        <rule object="SARepeatedCharacters">
+        <rule object="SARepeatedCharacters" tags="cosmetic,human,text">
```

Comment:
is this human or machine? I'd say machine, as this can be deterministically detected